x-pack/plugins/security_solution/server/lib/entity_analytics/asset_criticality/asset_criticality_data_client.ts

```diff
@@ -188,16 +201,30 @@ export class AssetCriticalityDataClient {
     }
   }
 
+  private getImplicitEntityFields = (record: AssetCriticalityUpsert) => {
```

Comment:
nit:
maybe this is also better to move to some helpers file? I think we already have one?